pyproject.toml

```diff
@@ -46,40 +48,50 @@ Source = 'https://github.com/pydantic/pydantic-extra-types'
 Changelog = 'https://github.com/pydantic/pydantic-extra-types/releases'
 # TODO: Documentation
 
-[tool.pytest.ini_options]
-testpaths = 'tests'
-filterwarnings = [
-    'error',
+[project.optional-dependencies]
```

Comment:
Either remove this, or remove `linting.in` and `testing.in`, I don't mind which.

Either way, these deps should have no version constraints, that's handled by the `.txt` files in requirements.